snippets/scripts/compass-github-importer/README.md

```diff
@@ -0,0 +1,82 @@
+## Bootstrap your Compass Catalog with from a Self-Hosted GitHub Instance
+
+This script
+1. Iterates over all your GitHub organizations and all the repos within them
+2. Creates Compass Components for each of them
```

Comment:
Apart from creating a component (with associated repo link), do we include any additional meta-data i.e repo description, labels etc?

Reply:
nvm I see below it includes "description", probably worth expanding on what import means in that sense i.e we create component and update description etc.